src/app/codegen-data-model/tests/TestCodegenModelViaMocks.cpp

```diff
@@ -2450,7 +2454,8 @@ TEST(TestCodegenModelViaMocks, EmberInvokeTest)
 
         const uint32_t kDispatchCountPre = chip::Test::DispatchCount();
 
-        ASSERT_EQ(model.Invoke(kInvokeRequest, tlvReader, nullptr /* handler, NOT used by impl*/), CHIP_NO_ERROR);
+        // Using a handler set to nullpotr as it is not used by the impl
```

Comment:
nit, nullptr

Reply:
```suggestion
        // Using a handler set to nullptr as it is not used by the impl
```

Commenting as suggestion so I can "merge" without checking out the branch myself :)